### ISSUE TITLE
The pretrained results are different from the paper

### ISSUE THREAD

**Issue by Yunxiao-Wang** [sunlightsgy/MEmoR#3]
I utilized the pretrained file 'train_test_14.pth' to do the test, but the result is unusually high. May I ask what is wrong with it?

test results:
`{'loss': 1.435765134438285, 'accuracy': 0.8762886597938144, 'macro_f1': 0.8011474609375, 'weighted_f1': 0.8669317472506151}`

config.json:
`{
    "name": "Train_14",
    "message": "MEmoR Baseline",
    "anno_file": "data/anno.json",
    "data_file": "data/data.json",
    "id_file": "data/train_id.txt",
    "emo_type": "fine_grained",
    "speakers": [
        "sheldon",
        "leonard",
        "howard",
        "rajesh",
        "penny",
        "bernadette",
        "amy",
        "others"
    ],
    "audio": {
        "feature_file": "data/features/audio_features.json",
        "feature_dim": 6373
    },
    "text": {
        "bert_model_name": "bert-large-uncased",
        "feature_file": "data/features/text_features.json",
        "feature_dim": 1024
    },
    "visual": {
        "frame_path": "data/clips_frames",
        "faces_feature_dir": "data/features/visual_features/face_features",
        "faces_names_dir": "data/features/visual_features/face_names",
        "obj_feature_dir": "data/features/visual_features/object_features",
        "env_feature_dir": "data/features/visual_features/environment_features",
        "dim_env": 2048,
        "dim_obj": 1230,
        "dim_face": 1024
    },
    "personality": {
        "used": true,
        "anno_file": "data/features/character_features.csv",
        "feature_dim": 118
    },
    "n_gpu": 1,
    "data_loader": {
        "type": "MEmoRDataLoader",
        "args": {
            "batch_size": 8,
            "shuffle": true,
            "validation_split": 0.2,
            "num_workers": 2,
            "seed": 2020
        }
    },
    "model": {
        "type": "AMER",
        "args": {
            "D_e": 128
        }
    },
    "optimizer": {
        "type": "Adam",
        "args": {
            "lr": 5e-05,
            "weight_decay": 0,
            "amsgrad": true
        }
    },
    "loss": "nll_loss",
    "metrics": [
        "accuracy",
        "macro_f1",
        "weighted_f1"
    ],
    "lr_scheduler": {
        "type": "StepLR",
        "args": {
            "step_size": 50,
            "gamma": 0.9
        }
    },
    "trainer": {
        "type": "MEmoRTrainer",
        "epochs": 1000,
        "save_dir": "saved/",
        "save_period": 50,
        "verbosity": 2,
        "monitor": "max val_accuracy",
        "early_stop": 50,
        "tensorboard": true
    }
}`

All other files remain unchanged.

**Comment by yizhouzhao** [comment]
Think is training results. Filter the dataset into testing.
